living_documentation_generator/utils/decorators.py

```diff
@@ -9,22 +30,26 @@
 
 
 def debug_log_decorator(method: Callable) -> Callable:
-    """
-    Decorator to add debug logging for a method call.
-    """
+    """Decorator to add debug logging for a method call."""
+
     @wraps(method)
     def wrapped(*args, **kwargs) -> Optional[Any]:
-        logger.debug("Calling method %s with args: %s and kwargs: %s.", method.__name__, args, kwargs)
+        logger.debug(
+            "Calling method %s with args: %s and kwargs: %s.",
+            method.__name__,
```

Comment:
Is here really needed to use 3 lines for arguments? I am not sure if it helps with readability.

Reply:
Since correct having correct setting of black tool, all these issues are gone.

Reply:
In the current version of code this debug call can be part of one line.

Reply:
Nice catch, fixed at: TDL